exchanges/coinbene/coinbene_wrapper.go

```diff
@@ -0,0 +1,369 @@
+		account.Currencies = append(account.Currencies,
+			exchange.AccountCurrencyInfo{CurrencyName: c,
+				TotalValue: hold + available,
+				Hold:       hold})
+	}
+	info.Accounts = append(info.Accounts, account)
+	info.Exchange = c.Name
+	return info, nil
+}
+
+// GetFundingHistory returns funding history, deposits and
+// withdrawals
+func (c *Coinbene) GetFundingHistory() ([]exchange.FundHistory, error) {
+	return nil, common.ErrFunctionNotSupported
+}
+
+// GetExchangeHistory returns historic trade data since exchange opening.
+func (c *Coinbene) GetExchangeHistory(p currency.Pair, assetType string) ([]exchange.TradeHistory, error) {
+	return nil, common.ErrFunctionNotSupported
+}
+
+// SubmitOrder submits a new order
+func (c *Coinbene) SubmitOrder(p currency.Pair, side exchange.OrderSide, orderType exchange.OrderType, amount, price float64, clientID string) (exchange.SubmitOrderResponse, error) {
+	var resp exchange.SubmitOrderResponse
+	if side != exchange.BuyOrderSide && side != exchange.SellOrderSide {
+		return resp, fmt.Errorf("%s orderside is not supported by this exchange", side)
+	}
+	tempResp, err := c.PlaceOrder(price, amount, exchange.FormatExchangeCurrency(c.Name, p).String(), orderType.ToString(), "")
+	if err != nil {
+		return resp, err
+	}
+	resp.IsOrderPlaced = true
+	resp.OrderID = tempResp.OrderID
+	return resp, nil
+}
+
+// ModifyOrder will allow of changing orderbook placement and limit to
+// market conversion
+func (c *Coinbene) ModifyOrder(action *exchange.ModifyOrder) (string, error) {
+	return "", common.ErrFunctionNotSupported
+}
+
+// CancelOrder cancels an order by its corresponding ID number
+func (c *Coinbene) CancelOrder(order *exchange.OrderCancellation) error {
+	_, err := c.RemoveOrder(order.OrderID)
+	return err
+}
+
+// CancelAllOrders cancels all orders associated with a currency pair
+func (c *Coinbene) CancelAllOrders(orderCancellation *exchange.OrderCancellation) (exchange.CancelAllOrdersResponse, error) {
+	var resp exchange.CancelAllOrdersResponse
+	orders, err := c.FetchOpenOrders(exchange.FormatExchangeCurrency(c.Name, orderCancellation.CurrencyPair).String())
+	if err != nil {
+		return resp, err
+	}
+	for x := range orders.OpenOrders {
+		_, err := c.RemoveOrder(orders.OpenOrders[x].OrderID)
+		if err != nil {
+			resp.OrderStatus[orders.OpenOrders[x].OrderID] = "Failed"
+		} else {
+			resp.OrderStatus[orders.OpenOrders[x].OrderID] = "Success"
+		}
+	}
+	return resp, nil
+}
+
+// GetOrderInfo returns information on a current open order
+func (c *Coinbene) GetOrderInfo(orderID string) (exchange.OrderDetail, error) {
+	var resp exchange.OrderDetail
+	tempResp, err := c.FetchOrderInfo(orderID)
+	if err != nil {
+		return resp, err
+	}
+	resp.Exchange = c.Name
+	resp.ID = orderID
+	resp.CurrencyPair = currency.NewPairWithDelimiter(tempResp.Order.BaseAsset, "/", tempResp.Order.QuoteAsset)
+	timestamp := tempResp.Order.OrderTime
+	resp.OrderDate = time.Unix(timestamp, 0)
```

Comment:
According to the docs fetching orderids is returned like ` "orderTime": "2019-06-12T09:39:27.0Z"` and you don't do any conversion in the API implementation. This will die  🐹💥🔨  You'll need to update your struct to use a string and then use `time.Parse(stringerino)` instead